.github/workflows/borg_integration_tests.yml

```
@@ -0,0 +1,282 @@
+            # For latest version
+            brew install borgbackup
+          fi
+          
+          # Verify installation
+          borg --version
+
+      - name: Install Docker via Homebrew
+        run: |
+          # Install Docker Desktop
+          HOMEBREW_NO_AUTO_UPDATE=1 brew install --cask docker
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)